src/pages/home/sidebar/SidebarScreen/BaseSidebarScreen.js

```diff
@@ -112,6 +112,7 @@ class BaseSidebarScreen extends Component {
                                 onAvatarClick={this.navigateToSettings}
                                 isSmallScreenWidth={this.props.isSmallScreenWidth}
                                 isDrawerOpen={this.props.isDrawerOpen}
+                                currentlyViewedReportID={this.props.currentlyViewedReportID}
```

Comment:
Passing this here is enough to ensure the `SidebarLinks` update with the `currentlyViewedReportID`. That might not be obvious though so going to add a comment I think.